src/MainnetController.sol

```diff
@@ -248,5 +272,37 @@ contract MainnetController is AccessControl {
         );
     }
 
+    /**********************************************************************************************/
+    /*** Relayer bridging functions                                                             ***/
+    /**********************************************************************************************/
+
+    function transferUSDCToCCTP(uint256 usdcAmount, uint32 destinationDomain)
+        external onlyRole(RELAYER) isActive
+    {
+        bytes32 mintRecipient = mintRecipients[destinationDomain];
+
+        require(mintRecipient != 0, "MainnetController/invalid-mint-recipient");
```

Comment:
I think a more descriptive error is `MainnetController/domain-not-configured`.

Reply:
Yeah good call, updated